lib/generators.js

```diff
@@ -346,6 +344,28 @@ function _getPathParamsFromKeys(keys) {
     return params;
 }
 
+/**
+ * this is used to create param for custom controllers
+ * @param parameters
+ * @returns {Array}
+ * @private
+ */
+function _getQueryParamsFromParameters(parameters) {
+    var params = [];
+
+    _.forEach(parameters, function (parameter) {
+        params.push({
+            in: parameter.in || 'query',
```

Comment:
here it is, running default as `query` it would make this process implicit, since parameter can perform both body and query param